### ISSUE TITLE
Koffi requires a webpack config workaround when using yarn workspaces

### ISSUE THREAD

**Issue by jackerman88** [Koromix/koffi#124]
Because koffi has a `require()` call with passed-in variables (at https://github.com/Koromix/rygel/blob/b5eeb955e03536e1c955c78b3159d682aa4bfb9b/src/koffi/index.js#L69-L91 ), webpack creates a context ( https://webpack.js.org/guides/dependency-management/#require-with-expression ) with a directory of `node_modules` and a regex including `koffi`. This results in every file of the koffi package (including readmes, changelogs, etc.) attempting to be processed by webpack which results in a bunch of errors.

I've been able to workaround this by using the tests in the electron-forge example:
```
// Add support for native node modules
  {
    // We're specifying native_modules in the test because the asset relocator loader generates a
    // "fake" .node file which is really a cjs file.
    test: /native_modules[/\\].+\.node$/,
    use: 'node-loader',
  },
  {
    test: /[/\\]node_modules[/\\].+\.(m?js|node)$/,
    parser: { amd: false },
    use: {
      loader: '@vercel/webpack-asset-relocator-loader',
      options: {
        outputAssetBase: 'native_modules',
      },
    },
  },
```

In conjunction with the ContextReplacementPlugin:
`new ContextReplacementPlugin(/node_modules/, /koffi\/index\.js/)`
which redirects any contexts with a `node_modules` directory to only include the koffi `index.js` file. This works for my use-case because no other libraries I use end up with a webpack context with a `node_modules` directory, but if ever I need to add one, it would cause this to break.

Is there anything that can be done on koffi's side to workaround this, or is this something webpack would need to do?

**Comment by Koromix** [comment]
I've pushed a possible workaround in the latest commit.

Could you try replacing koffi's index.js file with this version:
https://raw.githubusercontent.com/Koromix/rygel/9a56c3f39558086e9e385bf07b4d28595c08b2e9/src/koffi/index.js 

And report to me if it does fix your issue?

**Comment by Koromix** [comment]
I've published Koffi 2.7.2 with this change, hopefully it helps ;)
If not, leave a comment to reopen the bug!
